src/screens/ExchangeModal.js

```diff
@@ -517,6 +517,89 @@ export default function ExchangeModal({
     }
   };
 
+  const submit = useCallback(
+    async amountInUSD => {
+      setIsAuthorizing(true);
+      let NotificationManager = ios ? NativeModules.NotificationManager : null;
+      try {
+        const wallet = await loadWallet();
+        if (!wallet) {
+          setIsAuthorizing(false);
+          logger.sentry(`aborting ${type} due to missing wallet`);
```

Comment:
can set submitting to false if something happens with `loadWallet` and it didn't load? i manually made it fail and the button is stuck in `Submitting`

Reply:
probably similar to `setIsAuthorizing`

Reply:
nvm